content/en/docs/glossary.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Glossary
+description: A list of Notary Project's terminologies 
+weight: 11
+---
+### Notary 
+Notary is a TUF [(The Update Framework)](https://theupdateframework/) based artifact signing **implementation**. It consists of a server and client components. The server infrastructure tightly integrates with the container registry and keeps track of the images pushed to the registry. The client component is referred to as the notary command-line interface (CLI), which handles tasks such as signing and pushing container images to a registry and updating metadata. The CLI communicates with the registry and the key and metadata management server component. The most prominent use of this implementation is in Docker Content Trust (DCT). The server and the client implementation can be found in the [notary](https://github.com/notaryproject/notary) repository under the Notary Project organization.
+### Notation
+Notation, based on new set of specifications unrelated to TUF, is a newer implementation from the Notary Project. It refers to the implementation of the Notation CLI and the Notation libraries. Notation offers signing and verification workflows for OCI artifacts, enables signature portability across OCI-compliant registries, and facilitates integration with third-party key management solutions through a plugin model. The CLI itself is referred to as the [Notation CLI](https://github.com/notaryproject/notation) while the libraries are referred to as the [Notation-Go](https://github.com/notaryproject/notation-go) library and the [Notation-Core-Go](https://github.com/notaryproject/notation-core-go) library. Stakeholders can use libraries independently of the Notation CLI for integrating Notation with their own client tools.
+
+### Notary Project 
```

Comment:
Suggest moving Notary Project as the first glossary entry to set overall context and as it is referenced in Notation enrty.